src/address.rs

```diff
@@ -608,7 +610,7 @@ impl<'a> fmt::Display for AddressEncoding<'a> {
 /// ### Relevant BIPs
 ///
 /// * [BIP13 - Address Format for pay-to-script-hash](https://github.com/bitcoin/bips/blob/master/bip-0013.mediawiki)
-/// * [BIP16 - Pay to Script Hash](https://github.com/bitcoin/bips/blob/master/bip-0016.mediawiki)
+/// * [BIP16 - Pay to ScriptBuf Hash](https://github.com/bitcoin/bips/blob/master/bip-0016.mediawiki)
```

Comment:
Should be `Script` still.

Reply:
LOL, I did search&replace and I didn't realize the compiler won't complain if the doc is wrong.